src/Stryker.Core/Stryker.Core/TestRunners/VsTest/VsTestRunner.cs

```diff
@@ -507,7 +509,11 @@ private string GenerateRunSettings(int? timeout, bool forMutantTesting, bool for
                 settingsForCoverage += "<DisableParallelization>true</DisableParallelization>";
             }
             // TODO: get proper timeout
-            var timeoutSettings = timeout!=null ? $"<TestSessionTimeout>{timeout}</TestSessionTimeout>" + Environment.NewLine : string.Empty;
+            var timeoutSettings = timeout != null ? $"<TestSessionTimeout>{timeout}</TestSessionTimeout>" + Environment.NewLine : string.Empty;
+
+            var testCaseFilter = string.IsNullOrWhiteSpace(_options.TestCaseFilter) ?
+                string.Empty : $"<TestCaseFilter>{_options.TestCaseFilter}</TestCaseFilter>" + Environment.NewLine;
```

Comment:
I see potential issues here:
1. Should the test case filter syntax be escaped to fit into the XML? I don't know the complete specs for the option, but could complex expression break the XML file, such as: `(FullyQualifiedName~UnitTest1&TestCategory=CategoryA)|Priority=1`?
2. Could there an XML injection attack (or by mistake) if the user typed options like `--filter </RunSetting><AnotherOption />`? Should this be protected against in the user input `Validate()` call? How would you go about this?

Reply:
1. I don't know either. I guess we'll find out ;)
2. Since we don't process the xml there is no xml injection attack from our point of view. However I don't know if vstest protects against this. IMO it is not our job to protect vstest against injection attacks. If it is a risk, vstest needs to protect against this.

Reply:
Fair enough. I just wanted to make sure I wasn't missing something.